etc/profile-a-l/gnome-logs.profile

```diff
@@ -39,7 +39,7 @@ disable-mnt
 private-bin gnome-logs
 private-cache
 private-dev
-private-etc
+private-etc @sound
```

Comment:
```suggestion
private-etc machine-id
```

`@sound` seems rather confusing, as this has `nosound` and it only really needs
`machine-id` for the fix.

Either way, this likely conflicts with #5645 (which also fixes the issue), so
I'd wait for that instead.

Reply:
I don't have any problem waiting for #5645 to fix this. If it would make that any easier, I can close here. Feel free to suggest what you think is best.

Reply:
@glitsj16 [on Feb 11](https://github.com/netblue30/firejail/pull/5651#discussion_r1103710300):

> I don't have any problem waiting for #5645 to fix this.

:+1:

> If it would make that any easier, I can close here. Feel free to suggest what
> you think is best.

I don't think it needs closing, but to note the conflict I think it would make
sense to mark it as a draft.